pkg/nvlib/device/device.go

```diff
@@ -219,6 +219,9 @@ func (d *device) IsFabricAttached() (bool, error) {
 		if ret != nvml.SUCCESS {
 			return false, fmt.Errorf("error getting GPU Fabric Info: %v", ret)
 		}
+		if info.ClusterUuid == [16]uint8{} || info.CliqueId == 0 {
+			return false, nil
+		}
```

Comment:
Should this come after verifying that `GPU_FABRIC_STATE_COMPLETED`?

Reply:
Yes, it probably should.